config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  
+  resources :properties
```

Comment:
we will probs be leaving out a couple methods of resources so we wanna use the :except keyword after to remove those (just so hitting the route doesn't give a client side error rather than a not found)